kie-wb-deployment-validation/.gitignore

```diff
@@ -0,0 +1 @@
+/target/
```

Comment:
(Nitpick) Is this not already handled by [.gitignore](https://github.com/kiegroup/kie-wb-distributions/blob/master/.gitignore#L1) in the root of this project?

Reply:
I think this was auto-generated by Eclipse.

Reply:
Took a look and the project root has an absolute path, so it doesn't match targets of subprojects.